src/selectivity-templates.js

```diff
@@ -278,6 +278,9 @@ Selectivity.Templates = {
      *                mode - Mode in which select exists, single or multiple.
      */
     selectCompliance: function(options) {
+        if (options.mode === 'multiple' && !options.name.match(/\[\]$/)) {
```

Comment:
Why not use `options.name.slice(-1) !== '[]'`? That seems a bit easier for readability (and performance) than trying to parse a regular expression :)

Also, you can shorten the next line to `options.name += '[]'`.